database/hall2operations.py

```diff
@@ -333,10 +333,11 @@ def _translation(self, N):
         return T
 
     def _change_of_basis_symbol(self, V):
-        if V[0] == '-':
-            return np.array([0, 0, -1])
-        else:
-            return np.array([0, 0, 1])
+        # if V[0] == '-':
+        #     return np.array([0, 0, -1])
+        # else:
+        #     return np.array([0, 0, 1])
+        return np.array([0, 0, int(V[:-1])])
```

Comment:
Please don't comment code, just replace it

Reply:
I have removed them.